frontend/components/dashboard/table/SummaryDetails.vue

```diff
@@ -1,60 +1,77 @@
 <script setup lang="ts">
 import type { VDBSummaryTableRow } from '~/types/api/validator_dashboard'
```

Comment:
Another example here, it is very unclear what are 16 and 42 here:
![image](https://github.com/gobitfly/beaconchain/assets/150015231/9c95cbbb-32c0-46f3-b75f-b43741ebc31f)
16 and 42 do not add up to the "past" number.
So what are they?

Reply:
the 
`green / red ` scheme is used for `good / bad`through out the UI ... feel free to discuss this with Inan at one point (but this would effect the whole application) .

The numbers don't add up because they are randomly mocked by the backend